tests/renderStatsCard.test.js

```diff
@@ -326,7 +326,7 @@ describe("Test renderStatsCard", () => {
 
     document.body.innerHTML = renderStatsCard(stats);
```

Comment:
Maybe split this into two tests so that developers know what behaviour they break:
- should show short (i.e. abbreviated) numbers by default.
- should show longer numbers when 'number_format' is `long`.

Reply:
Okay, I will implement this when I get home

Reply:
Amazing looks good to me now 👍🏻. Thanks again for your contribution.